dev/Deployment/DeploymentManager.cpp

```diff
@@ -358,12 +355,12 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
         // it can exit or crash.  This API will be able to detect the failure and return.
         wil::handle_wait(processInfo.hProcess);
 
-        DWORD processExitCode;
-        GetExitCodeProcess(processInfo.hProcess, &processExitCode);
-        return HRESULT_FROM_WIN32(processExitCode);
+        DWORD processExitCode{};
+        THROW_IF_WIN32_BOOL_FALSE(GetExitCodeProcess(processInfo.hProcess, &processExitCode));
```

Comment:
SUGGESTION: `THROW_IF_WIN32_BOOL_FALSE_MSG(..., "%ls", cmdLine.get());`

If this fails and WIL's logging is captured you'll see the error, but on a running system it can be hard to idenitfy together what what this exitcode came from. That data context in the MSG would be invaluable to understanding logs ~if~ when the need arises